kola/tests/kubeadm/checksums.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -euo pipefail
+
+fail() {
+    IFS=' ' echo "$*" >&2
+    exit 1
+}
+
+# no limit by default
+limit=''
+if [[ ${#} -gt 0 ]]; then
+    if [[ ! "${1}" =~ ^-[0-9]+$ ]]; then
+        fail 'invalid flag to limit k8s versions, must be in form of -<number>'
+    fi
+    limit="${1//-/}"
+fi
+
+tuples=(
+    "KubeadmSum^https://storage.googleapis.com/kubernetes-release/release/%K8S_VER%/bin/linux/%ARCH%/kubeadm"
+    "KubeletSum^https://storage.googleapis.com/kubernetes-release/release/%K8S_VER%/bin/linux/%ARCH%/kubelet"
+    "CRIctlSum^https://github.com/kubernetes-sigs/cri-tools/releases/download/%CRICTL_VER%/crictl-%CRICTL_VER%-linux-%ARCH%.tar.gz"
+    "CNISum^https://github.com/containernetworking/plugins/releases/download/%CNI_VER%/cni-plugins-linux-%ARCH%-%CNI_VER%.tgz"
+    "KubectlSum^https://storage.googleapis.com/kubernetes-release/release/%K8S_VER%/bin/linux/%ARCH%/kubectl"
+)
+
+this_dir=$(dirname $0)
```

Comment:
```suggestion
this_dir=$(dirname "$0")
```
Shouldn't be a problem normally but who knows ;)

Reply:
Not planning to run it in `C:\Program Files (x86)\…` any time soon, but right you are. :)

Reply:
Will fix it after the docker image is built, so I can run the tests on Jenkins.

Reply:
Nevermind, fixed now. Also did a rebase, so the new docker image will include the extra tests for cilium.